src/routes/index.svelte

```diff
@@ -168,6 +169,18 @@
 	};
 
 	onMount(() => {
+		const darkMode = () => {
+			const dark: boolean = window.matchMedia('(prefers-color-scheme: dark').matches;
```

Comment:
Also, should we check if `window.matchMedia` exists?

Reply:
Yeah, you're right. Fixing.